docs/gcloud-auth.rst

```diff
@@ -58,15 +58,68 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful if your
```

Comment:
Any chance we can follow semantic line-breaks  ? (http://rhodesmill.org/brandon/2012/one-sentence-per-line/)